src/Monomer/Widgets/Singles/TextArea.hs

```diff
@@ -254,6 +264,7 @@ makeTextArea !wdata !config !state = widget where
   !caretMs = fromMaybe defCaretMs (_tacCaretMs config)
   !maxLength = _tacMaxLength config
   !maxLines = _tacMaxLines config
+  !editable = not (fromMaybe False (_tacReadOnly config))
```

Comment:
Alternatively you can do:
```haskell
!editable = _tacReadOnly /= Just True
```

Reply:
Indeed, that is neater: I just pushed the change.